### ISSUE TITLE
Setting loadBalancerIP on ingress

### ISSUE THREAD

**Issue by xeor** [banzaicloud/istio-operator#194]
**Is your feature request related to a problem? Please describe.**
It looks like there is no way currently to set `loadBalancerIP` on the ingress gateway..

**Describe the solution you'd like to see**
A way to set `loadBalancerIP` in the yaml under `gateways.ingress.loadBalancerIp`

**Describe alternatives you've considered**
Setting the ip as:

```
   gateways:
     ingress:
       serviceAnnotations:
         loadBalancerIP: 10.1.2.3
```
but that does nothing.. :)

**Comment by Laci21** [comment]
Hi @xeor, thank you for your feature request!

We've added the capability to set the `loadBalancerIP` for the ingress gateway (see #198). You can do it like this:
```
gateways:
  ingress:
    loadBalancerIP: 10.1.2.3
```

This feature will be available in the next operator release (0.1.13) which will be released by tomorrow. Let us know if it works for you and if you have any other idea to improve the operator.

**Comment by xeor** [comment]
Wow.. that was quick! Thanks a lot :)
